test/test_cache.py

```diff
@@ -115,6 +115,12 @@ def test_waypoints(self):
     def test_state(self):
         self.assertEqual(self.c.state, True)
 
+    def test_status(self):
```

Comment:
These tests seem to cover only the basic functionality. I would prefer if we would have a dedicated method which checks the correct parsing for all the available status values (except *unpublished* as this is hard to record/check).

Reply:
Added in https://github.com/tomasbedrich/pycaching/pull/193/commits/b9a643fc43ba795e689cb32587d1dbda5c5096cd